chore(kno-13473): clarify branch selector location (#1469)
knocklabs/docs: content/version-control/branches.mdx

**In the dashboard**

> You can create a new branch in the Knock dashboard by going to the **Branches** page in your account settings and clicking the "Create branch" button. Alternatively, you can type into the branch selector at the top of the Knock dashboard (directly to the right of the environment switcher) to quickly create a new branch.

Reviewer: ```suggestion
You can create a new branch in the Knock dashboard by going to the **Branches** page in your account settings and clicking the "Create branch" button. Alternatively, you can type into the branch selector at the top of the Knock dashboard to quickly create a new branch. By default, you'll be on the `main` branch.
```

Reviewer: I think we can pull back a bit here since you've added the figure below, but I do think it would be helpful to explain what it says by default. I imagine Chris was probably looking for something that said "Branch" and didn't think to look for "main" somewhere around the top of the page. (Though I agree our prior wording also sounded like you should look for this further left.)

Just a suggestion -- I trust your judgement if you want to leave as is.
